### PR TITLE
New package : dolphin-plugins-20.04.2

### PR DESCRIPTION
**Pull request by pandom79**
add shlibs

### CODE REVIEW

File: srcpkgs/dolphin-plugins/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools dolphin"
+makedepends="qt5-devel kxmlgui-devel kio-devel ktexteditor-devel dolphin"
+depends="dolphin"
+short_desc="These plugins integrate Dolphin with the revision control systems Bazaar, Mercurial and Git"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="GPL-2.0+"
```
Comment (Johnnynator) [inline srcpkgs/dolphin-plugins/template:12]:
```suggestion
license="GPL-2.0-or-later"
```

File: srcpkgs/dolphin-plugins/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'dolphin-plugins'
+pkgname=dolphin-plugins
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools dolphin"
+makedepends="qt5-devel kxmlgui-devel kio-devel ktexteditor-devel dolphin"
+depends="dolphin"
+short_desc="These plugins integrate Dolphin with the revision control systems Bazaar, Mercurial and Git"
```
Comment (Johnnynator) [inline srcpkgs/dolphin-plugins/template:10]: Use smth shorter like
`Plugins to integrate Dolphin with various revision control systems`.

File: srcpkgs/dolphin-plugins/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'dolphin-plugins'
+pkgname=dolphin-plugins
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```
Comment (Chocimier) [inline srcpkgs/dolphin-plugins/template:6]: Remove, we want tests, even if no exist for now.

File: srcpkgs/dolphin-plugins/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools dolphin"
```
Comment (Chocimier) [inline srcpkgs/dolphin-plugins/template:7]: kcoreaddons, kconfig instead of dolphin

File: srcpkgs/dolphin-plugins/template [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+makedepends="qt5-devel kxmlgui-devel kio-devel ktexteditor-devel dolphin"
+depends="dolphin"
+short_desc="These plugins integrate Dolphin with the revision control systems Bazaar, Mercurial and Git"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="GPL-2.0+"
+homepage="https://kde.org/applications/en/system/org.kde.dolphin_plugins"
+distfiles="${KDE_SITE}/release-service/20.04.2/src/${pkgname}-${version}.tar.xz"
```
Comment (Chocimier) [inline srcpkgs/dolphin-plugins/template:14]: use ${version}

**Comment by Chocimier** [comment]
Looks good. Please [fixup](https://git-scm.com/docs/git-rebase#_interactive_mode) commits into one, named as per CONTRIBUTING.md.

**Comment by pandom79** [comment]
> Looks good. Please [fixup](https://git-scm.com/docs/git-rebase#_interactive_mode) commits into one, named as per CONTRIBUTING.md.

done

**Pull request merged**